### PR TITLE
Added RFID to students with functions and tests [tff-208]

### PR DESCRIPTION
**Pull request by Cafferoq**
### Summary

Added RFID to student objects, fixed functional tests. Added access functions and unit tests.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Make sure code looks good and run tests.

### Links

- [TFF-208](https://msoese.atlassian.net/browse/TFF-208)

### CODE REVIEW

File: src/store/student-store.js [hunk 1]
```diff
@@ -7,7 +6,7 @@ import CheckinStore from './checkin-store';
 import ModelStore from './model-store';
 import { isBeforeNow, dueDateToTimestamp } from '../lib/clock';
 
-const ACTIVE_STATUS = config.get('student.active_status');
+const ACTIVE_STATUS = 'C - Current';
```
Comment (gearedmonkey) [inline src/store/student-store.js]: Why did you remove this as a config?

Comment (Cafferoq) [inline src/store/student-store.js]: That wasn't me... git's been commiting weird things on my machine. I'll remove that from this PR
